core/src/apps/nem/get_address.py

```diff
@@ -24,16 +24,19 @@ async def get_address(msg: NEMGetAddress, keychain: Keychain) -> NEMAddress:
     network = msg.network  # local_cache_attribute
 
     validate_network(network)
-    await validate_path(keychain, address_n, check_path(address_n, network))
+    await paths.validate_path(keychain, address_n, check_path(address_n, network))
 
     node = keychain.derive(address_n)
     address = node.nem_address(network)
 
     if msg.show_display:
+        from . import PATTERNS
+
         await show_address(
             address,
             case_sensitive=False,
-            path=address_n_to_str(address_n),
+            path=paths.address_n_to_str(address_n),
+            account=f"NEM #{paths.get_account_num(msg.address_n, list(PATTERNS))}",
```

Comment:
Nit: if the function accepted `Sequence[...]` instead of `list[...]` there would be no need to convert to list.

Reply:
Right, changed in [b558b39](https://github.com/trezor/trezor-firmware/pull/3299/commits/b558b39a7128223d023b95140d0257bf93e1261f)